### PR TITLE
Media 3.0.0 release (staging -> main)

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,43 @@
 # Adobe Experience Platform - Media extension for Android
 
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/adobe/aepsdk-media-android/tree/dev-v2%2E0%2E0.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/adobe/aepsdk-media-android/tree/dev-v2%2E0%2E0)
-[![codecov](https://codecov.io/gh/adobe/aepsdk-media-android/branch/main/graph/badge.svg?token=UgtU9vDA6j)](https://codecov.io/gh/adobe/aepsdk-media-android)
+[![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/media.svg?logo=android&logoColor=white&label=media)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/media)
```
Comment (addb) [inline README.md:3]: Should we add coverage and build tags?

File: code/testapp/src/main/assets/ADBMobileConfig.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
 	"global.privacy": "optedin",
 	"analytics.launchHitDelay": 0,
 	"audience.timeout": 5,
-	"media.trackingServer": "pst-beta.hb-api.omtrdc.net",
+	"media.trackingServer": "obumobile5.hb.omtrdc.net",
```
Comment (addb) [inline code/testapp/src/main/assets/ADBMobileConfig.json]: Let's remove this value as latest SDK doesn't read it.

File: code/testapp/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
         android:cursorVisible="true"
         android:gravity="left"
         android:hint="Enter Assurance Url"
-        android:text="mediaapp://?adb_validation_sessionid=466dbe70-f864-4104-8121-d844dc34d179" />
+        android:text="" />
```
Comment (addb) [inline code/testapp/src/main/res/layout/activity_main.xml:26]: remove this?

File: README.md (hunk 1 above)
Comment (praveek) [reply README.md:3]: No. We decided not to add those in Readme.md

File: code/testapp/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (praveek) [reply code/testapp/src/main/res/layout/activity_main.xml:26]: Yes. Removed hard-coded value for the Assurance session.

**Review by addb — APPROVED** [review]

**Pull request merged**